### PR TITLE
Fish Trawl Survey

### PR DESCRIPTION
**Pull request by mcmcgrath13**
Add the fish trawl summary and dashboard.  Note, the preview won't work until the corresponding API PR is deployed.

Currently, the text in the summary is lifted from the URI website - we'll want them to bless that before publishing.  I'll pass the preview link to Jeff for review before deploying.

### CODE REVIEW

**Review by eldu — COMMENTED** [review]
I'm assuming that the corresponding API PR is in another repo? I would definitely love to take another look when that goes through as well.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+=======
+# Narragansett Bay Data Explorer
+
+A web app to explore data about the Narragansett bay.  Built using Nuxt (vue).  Uses the buoy-api as a backend.
+>>>>>>> feat: stub out fish trawl
+=======
+# Narragansett Bay Data Explorer
+
+A web app to explore data about the Narragansett bay.  Built using Nuxt (vue).  Uses the buoy-api as a backend.
+>>>>>>> 86ce104aa065d2883d91a4e5ce566cb6ed2b3862
```
[the remark this answers is not in the document]
Comment (mcmcgrath13) [reply README.md]: oh wow - what even happened here...

**Comment by mcmcgrath13** [comment]
> I'm assuming that the corresponding API PR is in another repo? I would definitely love to take another look when that goes through as well.

yup! https://github.com/ridatadiscoverycenter/buoy-api/pull/15

File: store/fish.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      { root: true }
+    );
+  },
+  async fetchSamples({ commit }) {
+    const samples = await this.$axios.$get(`/${route}/species`);
+    let species = samples.map(({title}) => title).sort()
+    species = species.filter((v, i, a) => a.indexOf(v) === i);
+    commit('mutate', { property: 'species', with: species })
+    const dataParsed = samples.map(d => {
+      if (d.species.includes('crab')){
```
Comment (eldu) [inline store/fish.js:35]:
```suggestion
      if (d.species.toLowerCase().includes('crab')){
```

Suggestion to match the rest of this if branching. Should `d.species.toLowerCase()` be saved in a temp variable?

**Review by eldu — COMMENTED** [review]
I look a look at this the other day and it looked really good ! I wasn't sure what sort of review to add in. 

Maybe another thing to think about is making the blank data and zero data more obvious at a glance. The tooltip when hovering that shows blank data or zero data is great and then also what about at a glance looking at the chart as a whole?

File: README.md [hunk 3]
```diff
@@ -1,9 +1,12 @@
+
 # [Narragansett Bay Data Explorer](https://data-explorer.riddc.brown.edu)
 
 [![DOI](https://zenodo.org/badge/271652303.svg)](https://zenodo.org/badge/latestdoi/271652303)
 
 > Explore data from the [RI Data Discovery Center](https://ridatadiscovery.org) about the Narragansett Bay.
 
+A web app to explore data about the Narragansett bay.  Built using Nuxt (vue).  Uses the buoy-api as a backend.
```
Comment (mcmcgrath13) [inline README.md:8]:
```suggestion
A web app to explore data about the Narragansett bay.  Built using Nuxt (Vue).  Uses the buoy-api as a backend.
```

**Pull request merged**